app/src/pages/inside/dashboardItemPage/modals/common/widgetControls/controls/filtersControl/filtersControl.jsx

```diff
@@ -134,6 +134,9 @@ export class FiltersControl extends Component {
   }
 
   getFormAppearanceComponent = (activeFilter) => {
+    if (!activeFilter) {
+      this.clearFormAppearance();
```

Comment:
I don't think it's a good place for a side-effect here. This is just a getter function

Reply:
Fixed